.gitignore

```diff
@@ -25,5 +25,8 @@ erl_crash.dump
 .elixir_ls
 
 .elixir_tools
+grpc_client/.elixir-tools/
+grpc_core/.elixir-tools/
+grpc_server/.elixir-tools/
```

Comment:
This shouldn't be included in the library, but in the dev's global gitignore IMO
```suggestion
```